CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ Features, Bug Fixes, Breaking Changes, Deprecated
 
 - Used app account instead of `deposit_lsig` in `examples/dao`
 - Support RekeyTo field in the inner transaction for TEAL v6.
+- Default add `infrastructure/` folder when initial algob project. You can create algob project without `infrastructure/` by use `--no-infrastructure` flag.  Example:
+
+```base
+  algob init --no-infrastructure
+```
```

Comment:
I'm really wondering what should be the default... @ratik21  did you think more about it?

Reply:
> I'm really wondering what should be the default... @ratik21 did you think more about it?

What happened if developer use the third party API? We don't need `infrastructure` in this case. Developer only want change API key in algob.config.js file to private network, test net, main net. I think add infrastructure should be optional. How to think about it @ratik21 @robert-zaremba ?

Reply:
Right, i think it's OK to assume developer will move to testnet/mainnet. But probably everyone starts project development on a dev network (it's very fast).. So i think `infrastructure` should be copied by default

Reply:
Right, i think it's safe to assume user will move to testnet/mainnet. But i think probably everyone starts with dev network for development (it's very fast)..An alternative could be using api-as-a-service (purestake). But i think we should copy the `infrastructure` dir by default